frontend/public/actions/features.ts

```diff
@@ -170,6 +170,9 @@ const detectShowOpenShiftStartGuide = (dispatch, canListNS: boolean = false) =>
 const ssarChecks = [{
   flag: FLAGS.CAN_GET_NS,
   resourceAttributes: { resource: 'namespaces', verb: 'get' },
+}, {
+  flag: FLAGS.CAN_GET_SECRET,
```

Comment:
This should not be a global flag. You need to check the user's access in the current namespace with the `useAccessReview` hook.

Reply:
Switch to useAccessReview and remove this flag